processes/release_processes/README.MD

```diff
@@ -30,13 +34,17 @@ for any performance changes since the previous release.
 runs and fill in any missing entries in the table.
  
 
-## How to Deploy Cromwell in CAAS prod
+### How to Deploy Cromwell in CAAS prod
 
 **Note:** Full details on how to complete each of these steps is found in the [Quick CAAS Deployment Guide](https://docs.google.com/document/d/1s0YC-oohJ7o-OGcgnH_-YBtIEKmLIPTRpG36yvWxUpE) 
 
 ![caas-prod](caas-prod.dot.png)
 
-## How to Generate and Publish Swagger Client Library
+## Bonus Processes
+
+The swagger client library is not part of our core publish/release process but can be performed from time to time, as required.
+
+### How to Generate and Publish Swagger Client Library
```

Comment:
Can you explain what is Swagger Client Library? I am not sure how it is related to Cromwell release

Reply:
This was added by @kcibul I think - and I'm just shuffling it around. 

I think it's a way of creating swagger client libraries from the yaml definitions but it's still not part of the main release process (hopefully the new headings in this doc make that clear?)

Reply:
Ah I see. Thank you! :)